app.js

```diff
@@ -0,0 +1,10 @@
+var express = require('express')
+var app = express();
+
+app.get('/', function (req, res) {
+    res.send("Hello World")
+})
+
+app.listen('8080', function () {
+    console.log('App is listening on port 8080')
+})
```

Comment:
export the app as a module, and move this to a `server.js` file. so we can do something like

`node server.js` to start the server instead of `node app.js`